glue-codes/simulink/src/create_FAST_SFunc.m

```diff
@@ -39,13 +39,16 @@
 else    
 %% defaults for cmake builds:
 
-    if ( ispc ) % Windows PC
+    if ( ismac )  % Apple MacOS
+        installDir = '../../../install';
+        outDir = fullfile(installDir, 'lib');
+    elseif ( ispc ) % Windows PC
         installDir = '../../../install';
         outDir = fullfile(installDir, 'lib');
         % If there are shared libraries does it work for outDir to be the local directory?
     else
-        installDir = '/usr/local';
-        outDir = '.';
+        installDir = '../../../install';
+        outDir = fullfile(installDir, 'lib');
```

Comment:
Do we need this `if` statement anymore? Looks like the 3 cases are now identical. 

Also, I thought there was some issue if this wasn't installed in `/usr/local` on MacOS??? Or was that related to a different issue that is now fixed?

Reply:
This is a bit of a work in progress at the moment.

The recent Catalina update to MacOS (I got upgraded 2 weeks ago) has new security measures (hardened runtime environment) that don't allow programs such as Matlab access to the environment variable `LD_LIBRARY_PATH`, so Matlab can no longer find dylibs in `/usr/local`.  We also changed the default install location to `<build_dir>/../install/lib` at some point in the last few years.

Anyhow, this still didn't play well with Matlab, but can be worked around by adding `@loader_path/` to the `rpath` during compile of the libraries (currently testing `-DCMAKE_INSTALL_NAME_DIR=@loader_path -DCMAKE_MACOSX_RPATH=ON` at the cmake setup step, but @rafmudaf may have a better method).  I wasn't entirely sure that we didn't need to pass additional settings to mex compiler as well, so I left all three cases just in case we still have a few system specific tweaks to make to that (I'll try to remember to remove those redundancies at the PR if they are still there).